community/modules/compute/htcondor-execute-point/README.md

```diff
@@ -127,6 +127,37 @@ the University of Wisconsin-Madison. Support for HTCondor is available via:
 
 [chtc]: https://chtc.cs.wisc.edu/
 
+## Behavior of Managed Instance Group (MIG)
+
+Regional [MIGs][mig] are used to provision Execute Points. By default, VMs
+will be provisioned in any of the zones available in that region, however, it
+can be constrained to run in fewer zones (or a single zone) using
+[var.zones](#input_zones).
+
+By default, the VM replacement policy is set to [opportunistic]. In practice,
```

Comment:
1. I don't think this link goes anywhere
2. Shouldn't this be formatte similarly to the other uses of the words e.g. "OPPORTUNISTIC"
3. Maybe the link should be in a sentence like: "There are two VM replacement policies [opportunistic or proactive].  By default the VM replacement ... "  The reason I like this is because it tells the user the link gives information on both and differentiates the use of "OPPORTUNISTIC" from the lowercase version.

The last two points apply to the other markdown files in this PR.

Reply:
Please check dbdca702efae96a432ae779611f1d0cfd2103874

Reply:
I also fixed another broken markdown link in 68d937cab629533e272b1d0c74aa82d17a23268b